gr-iio/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# Copyright 2011,2012 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+
+########################################################################
+# Setup dependencies
+########################################################################
+include(GrBoost)
+
+find_package(IIO)
+
+set(GR_IIO_DEPENDENCIES
+  Boost_FOUND
+  ENABLE_GNURADIO_RUNTIME
+  ENABLE_GR_BLOCKS
+  ENABLE_VOLK
+  IIO_FOUND)
+
+option(LIBAD9361_ENABLE "Build with libad9361-iio" ON)
+if (LIBAD9361_ENABLE)
```

Comment:
Thinking the default here (when `LIBAD9361_ENABLE` is not set on the commandline or elsewhere, and hence defaults to `ON`) should be to use `libad9361-iio` if it is found & otherwise don't use it. As written, the default if `libad9361-iio` is -not- found will cause this component to be disabled.

Reply:
Yes, you have to explicitly disable libad9361 for it to enable the module without it installed. To build without libad9361-iio by default would just require a if(LIBAD9361_FOUND) around the append. Is that the better default behavior?

Reply:
hmmm ... not quite. If `-DLIBAD9361_ENABLE=ON` is specified on the cmake commandline, then we always want to add `LIBAD9361_FOUND` to the `GR_IIO_DEPENDENCIES`. OTOH, if `LIBAD9361_ENABLE` is not specified one way or the other on the cmake commandline, then we want to add `LIBAD9361_FOUND` to the `GR_IIO_DEPENDENCIES` only if `LIBAD9361_FOUND` is TRUE.

Reply:
With that specification I don't think using the cmake OPTION command is useful since it cannot distinquish between user input and a default option.